### PR TITLE
Bug 2063756: - User Preferences - Applications - Insecure traffic : i18n misses 

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
@jhadvig /assign

### Before:
Screen Shot 2022-03-21 at 5 17 44 PM
Screen Shot 2022-03-21 at 5 17 52 PM

### After:
Screen Shot 2022-03-21 at 5 15 02 PM
Screen Shot 2022-03-21 at 5 15 12 PM

### CODE REVIEW

File: frontend/packages/dev-console/src/components/import/import-types.ts [hunk 1]
```diff
@@ -310,13 +310,18 @@ export enum TerminationTypes {
 }
 
 export enum InsecureTrafficTypes {
+  // t('devconsole~None')
   None = 'None',
+  // t('devconsole~Allow')
   Allow = 'Allow',
+  // t('devconsole~Redirect')
```
Comment (jhadvig) [inline frontend/packages/dev-console/src/components/import/import-types.ts:317]: Dont see `Allow` and `Redirect` translations in the devconsole locales. We should be adding them.

**Review by jhadvig — CHANGES_REQUESTED** [review]
Looks like we are missing locales for some of the strings

**Review by jhadvig — APPROVED** [review]
@cyril-ui-developer thank you for the fix!
/lgtm
/approve

**Comment by andrewballantyne** [comment]
@cyril-ui-developer There is a bug on this PR. When we translate the enums backing the dropdown items, we need to translate them using qualified namespaces otherwise it defaults to `public` translations. Pseudo translations only interact on the usage of `t` and not actually verify the translations are in the right qualified namespace.

Current Error on 4.11.0-0.nightly-2022-04-01-172551
image

I quickly hacked in an update to your code... 
The diff:
```
diff --git a/frontend/packages/dev-console/src/components/import/import-types.ts b/frontend/packages/dev-console/src/components/import/import-types.ts
index 0c1c5c77c4..4207b5bde6 100644
--- a/frontend/packages/dev-console/src/components/import/import-types.ts
+++ b/frontend/packages/dev-console/src/components/import/import-types.ts
@@ -305,27 +305,27 @@ export interface ImportData {

 export enum TerminationTypes {
   // t('devconsole~Edge')
-  edge = 'Edge',
+  edge = 'devconsole~Edge',
   // t('devconsole~Passthrough')
-  passthrough = 'Passthrough',
+  passthrough = 'devconsole~Passthrough',
   // t('devconsole~Re-encrypt')
-  reencrypt = 'Re-encrypt',
+  reencrypt = 'devconsole~Re-encrypt',
 }

 export enum InsecureTrafficTypes {
   // t('devconsole~None')
-  None = 'None',
+  None = 'devconsole~None',
   // t('devconsole~Allow')
-  Allow = 'Allow',
+  Allow = 'devconsole~Allow',
   // t('devconsole~Redirect')
-  Redirect = 'Redirect',
+  Redirect = 'devconsole~Redirect',
 }

 export enum PassthroughInsecureTrafficTypes {
   // t('devconsole~None')
-  None = 'None',
+  None = 'devconsole~None',
   // t('devconsole~Redirect')
-  Redirect = 'Redirect',
+  Redirect = 'devconsole~Redirect',
 }

 export interface AutoscaleWindowType {
```

**Pull request merged**